src/app/interactions/interactions-routing.module.ts

```diff
@@ -247,6 +248,11 @@ export const interactionsRoutes: Routes = [
         data: interactionsRoutesData["overlay-scroll-sample-2"],
         path: "overlay-scroll-sample-2"
     },
+    {
+        component: OverlayPresetSettingsSampleComponent,
+        data: interactionsRoutesData["overlay-relative-presets-sample"],
```

Comment:
the new sample was added with key `overlay-preset-settings-sample` in `interactionsRoutesData`. The key here is wrong.
```suggestion
        data: interactionsRoutesData["overlay-preset-settings-sample"],
```